core.js

```diff
@@ -57,8 +57,14 @@ class SiteMapper {
     isIgnoredPath(site) {
         let toIgnore = false;
         for (const ignoredPath of this.ignoredPaths) {
-            if (site.includes(ignoredPath))
-                toIgnore = true;
+            if (typeof ignoredPath == 'string') {
+                if (site.includes(ignoredPath))
+                    toIgnore = true;
+            }
+            else {
```

Comment:
I like your solution but I prefer to check if is a regex or not. If you can do it, it will be awesome.
You can use instanceof RegExp maybe.